lib/grammers-client/Cargo.toml

```diff
@@ -34,6 +34,7 @@ os_info = { version = "3.0.4", default_features = false }
 pin-project-lite = "0.2"
 pulldown-cmark = { version = "0.8.0", default-features = false, optional = true }
 tokio = { version = "1.5.0", features = ["sync", "fs", "macros", "time", "sync"] }
+anyhow = { version = "1.0.58" }
```

Comment:
So far the library has been doing a good job avoiding extra unnecessary dependencies. While this means a little more code to properly handle errors, I think it's worth it.